Modules/Server/Tests/PocketCastsServerTests/ServerHelperTests.swift

```diff
@@ -0,0 +1,125 @@
+
+        XCTAssertEqual(firstPodcastUpdates?.count, 2)
+        XCTAssertEqual(secondPodcastUpdates?.count, 1)
+
+        XCTAssertEqual(firstPodcastUpdates?[0].uuid, "d7dbe109-3436-4795-b608-5ef93a899d03")
+        XCTAssertEqual(firstPodcastUpdates?[0].url, "https://chrt.fm/track/GD6D57")
+        XCTAssertEqual(firstPodcastUpdates?[0].title, "Podcast Title")
+        XCTAssertEqual(firstPodcastUpdates?[0].episodeDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[0].detailedDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[0].fileType, "audio/mp3")
+        XCTAssertEqual(firstPodcastUpdates?[0].sizeInBytes, 0)
+        XCTAssertEqual(firstPodcastUpdates?[0].duration, 6210)
+        XCTAssertEqual(firstPodcastUpdates?[0].episodeType, "full")
+        XCTAssertEqual(firstPodcastUpdates?[0].seasonNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[0].episodeNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[0].publishedDate, "2024-01-26 14:42:16")
+
+        XCTAssertEqual(firstPodcastUpdates?[1].uuid, "99c63fd5-8ea5-438a-a3a9-60446af17794")
+        XCTAssertEqual(firstPodcastUpdates?[1].url, "https://chrt.fm/track/GD6D57/url")
+        XCTAssertEqual(firstPodcastUpdates?[1].title, "Podcast Title 2")
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[1].detailedDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[1].fileType, "audio/mp3")
+        XCTAssertEqual(firstPodcastUpdates?[1].sizeInBytes, 2271)
+        XCTAssertEqual(firstPodcastUpdates?[1].duration, 2271)
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeType, "full")
+        XCTAssertEqual(firstPodcastUpdates?[1].seasonNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[1].publishedDate, "2024-01-26 13:32:42")
+
+        XCTAssertEqual(secondPodcastUpdates?[0].uuid, "161160ef-fc5b-4165-b2a3-c52963733f0c")
+        XCTAssertEqual(secondPodcastUpdates?[0].url, "https://sphinx.acast.com/p/open/s/593eded1acfa040562f3480b/e/65ae84931a5c7e0017941cbc/media.mp3")
+        XCTAssertEqual(secondPodcastUpdates?[0].title, "Episode 877")
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeDescription, nil)
+        XCTAssertEqual(secondPodcastUpdates?[0].detailedDescription, nil)
+        XCTAssertEqual(secondPodcastUpdates?[0].fileType, "audio/mp3")
+        XCTAssertEqual(secondPodcastUpdates?[0].sizeInBytes, 89228708)
+        XCTAssertEqual(secondPodcastUpdates?[0].duration, 3605)
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeType, "full")
+        XCTAssertEqual(secondPodcastUpdates?[0].seasonNumber, 0)
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeNumber, 0)
+        XCTAssertEqual(secondPodcastUpdates?[0].publishedDate, "2024-01-26 15:00:52")
+    }
+}
+
+private extension ServerHelperTests {
+    func dataJSON() -> Data {
+        """
```

Comment:
I wasn't able to use this content inside a JSON file, so I just copied it here.

Reply:
I'm curious about this, why couldn't you read it from a file?

Reply:
I agree, can we move this to a file?

Reply:
Is just not that simple, unfortunately.

This test is on the `Server` module and that's what makes this complicated, I can't just simply access the file it as I would by testing on the app test target.

I can move it to a file and make some changes on the `Package.swift` but then the file will be copied as part of module, even though it's not needed by the app/module. See the first answer here: https://stackoverflow.com/questions/67662742/getting-file-in-path-spm-swift

I can try the second answer there. But it involves a lot of additional code for something that, in theory, should be simple.

What do we prefer? Keep this way? Copy it to the app even though is not needed? Try adding additional code for that?

Reply:
I did the additional code here, let me know what do you think: 60bb34050234693e5f9028c1f227e58dd593197e

Reply:
We can use `Bundle.module` for accessing resources in a Swift Package: https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package#Access-a-resource-in-code

The caveat here is that you need to have at least 1 file being copied to the test target for the code to be generated.